src/test/java/core/basesyntax/RegistrationValidationTest.java

```diff
@@ -0,0 +1,70 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationValidationTest {
+    private User testUser;
+
+    @BeforeEach
+    void setUp() {
+        testUser = new User();
+        testUser.setLogin("testLogin");
+    }
+
+    @Test
+    void get_existingUser_notOk() {
+        User user = new User();
+        user.setLogin("User999");
+        StorageDao storageDao = new StorageDaoImpl();
+        User storageUser = storageDao.get(user.getLogin());
+        boolean isExist = storageUser != null;
+        assertFalse(isExist, "User is already exists");
+    }
+
+    @Test
+    void setLogin_lessThanSixCharacters_notOk() {
+        testUser.setLogin("user");
+        String login = testUser.getLogin();
+        assertFalse(login.length() >= 6, "Login should be at least 6 characters");
+    }
+
+    @Test
+    void setPassword_lessThanSixCharacters_notOk() {
```

Comment:
You should test register method in service, so read the task description and see the comments above and improve your solution